salt/roots/worker/service.sls

```diff
@@ -0,0 +1,11 @@
+resque worker service:
+  cmd.run:
+    - name: 'nohup bundle exec rake resque:work > /vagrant/log/resque.log 2> /vagrant/log/resque.err < /dev/null &'
+    - cwd: /vagrant
+    - unless: pgrep -fa resque
+    - env:
+      - DB_HOST: 192.168.69.20
```

Comment:
It hasn't needed it in my experience. I don't see anywhere that uses that environment variable, either. Oh, it's part of the swipely/docker-api#86 library that we use. I'd rather address this in an additional feature (#20) of pulling more of this out of the states and into pillar data.